great_expectations/cli/cli.py

```diff
@@ -471,6 +472,91 @@ def profile(datasource_name, data_assets, profile_all_data_assets, directory, vi
         )
 
 
+@cli.command()
+@click.argument("data_asset_name")
+@click.argument("suite_name")
+@click.option(
+    '--batch-kwargs',
+    default=None,
+    help='Additional keyword arguments to be provided to get_batch when loading \
+    the data asset. Must be a valid JSON dictionary'
+)
+@click.option(
+    "--directory",
+    "-d",
+    default=None,
+    help="The project's great_expectations directory.",
+)
+@click.option(
+    "--jupyter-notebook",
+    "-jn",
+    default=True,
+    is_flag=True,
+    help="By default open in browser unless you specify the --no-jupyter flag",
+)
+@click.option(
+    "--jupyter-lab", "-jl", is_flag=True, default=False, help="Edit in jupyter lab",
+)
+@click.option(
+    "--jupyter/--no-jupyter",
+    is_flag=True,
+    help="By default open in browser unless you specify the --no-jupyter flag",
+    default=True,
+)
+def edit_suite(
+    data_asset_name, suite_name, directory, jupyter_notebook, jupyter_lab, jupyter, batch_kwargs
+):
+    """Edit an existing suite."""
+    try:
+        context = DataContext(directory)
+    except ge_exceptions.ConfigNotFoundError as err:
+        cli_message("<red>{}</red>".format(err.message))
+        return
+    except ge_exceptions.ZeroDotSevenConfigVersionError as err:
+        _offer_to_install_new_template(err, context.root_directory)
+        return
+
+    suite = _load_suite(context, data_asset_name, suite_name)
+
+    if not batch_kwargs:
+        # TODO rrrgh why aren't batch kwargs in a suite make this work...
+        cli_message("<red>This probably won't work.</red>")
+        batch_kwargs = context.yield_batch_kwargs(suite.data_asset_name)
+    else:
+        batch_kwargs = json.loads(batch_kwargs)
+
+    human_data_asset_name = suite.data_asset_name.generator_asset
```

Comment:
We might want to do some sort of check to make sure that this name doesn't exist, because it would be easy to have the same generator_asset name and end up overwriting yourself in a hard-to-understand way with this code.

Reply:
I'm not sure I follow. Note the human name is probably a cheap shortcut for the fully qualified name.